.github/workflows/comment-pending-deployment-info.yaml

```diff
@@ -0,0 +1,59 @@
+# Inspired from https://stefanbuck.com/blog/codeless-contributions-with-github-issue-forms
+name: Comment on issues with `new hub` label that are requesting a deployment with the pending information
+on:
+  issues:
+    types:
+      - labeled
+jobs:
+  ask-for-pending-information:
+    if: github.event.label.name == 'new hub'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Parse new hub request issue template form
+        uses: stefanbuck/github-issue-praser@v3
+        id: issue-parser
+        with:
+          template-path: .github/ISSUE_TEMPLATE/2_new-hub-provide-info.yml
+      - name: Find missing info in form
+        shell: python
+        id: find-missing-info
+        run: |
+          import os
+
+          form = ${{ steps.issue-parser.outputs.jsonString }}
+          # This will hold a string with all the pending information in the form
+          pending_info = ""
+          community_rep = form.get("community_representative_handle", None)
+
+          # Parse the form entries
+          for form_entry_id, form_entry_value in form.items():
+              # Add the form entry ids that were empty or marked as pending
+              # to a string, comma separated
+              if (
+                  form_entry_id != "tasks_to_deploy"
+                  and "optional" not in form_entry_id
+                  and (not form_entry_value or form_entry_value == "pending")
+              ):
+                  # Form ids have `_` in them, so replace their occurrences with a space
+                  # to improve readability
+                  pending_info += f'{form_entry_id.replace("_", " ")}, '
+          # Export the `pending_info` string to GITHUB_OUTPUT to be used in following steps
+          with open(os.environ["GITHUB_OUTPUT"], "a") as f :
+              print(f"pending_info={pending_info[:-2]}", file=f)
+              print(f"community_rep={community_rep}", file=f)
```

Comment:
I believe you will need to explicitly append a newline character `\n` here, otherwise both variables may end up on the same line and will not be parsable. I got bitten by this in another workflow.

Reply:
I echoed the two vars in a step below at some point, and fortunately they worked, so I think for some reason they are also ok  like this? 🤷🏼‍♀️

Reply:
Yeah, it might be because you used `print(..., file=...)` which I think would append a newline. But I was using `f.write()` which I don't think does.